finder.go

```diff
@@ -246,7 +248,7 @@ func tableFromStructType(structName string, structType *ast.StructType, packageP
 	return t, nil
 }
 
-func stringToDosaType(inType string, packagePrefix string) Type {
+func stringToDosaType(inType string) Type {
```

Comment:
Removing this means that when someone imports dosa and renames the package, it won't work correctly. Please add a test entity with dosa renamed that has the new types.